script/gettools.sh

```diff
@@ -6,6 +6,7 @@ gettools_develop() {
 	tools="github.com/bufbuild/buf/cmd/buf@v1.0.0-rc12
 google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
 google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2.0
+github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema@latest
```

Comment:
Is it json schema that is required or openapiv3? Not sure if there's a difference, but pretty sure the initial ticket asked for this?